content/environment_variables.tex

```diff
@@ -31,13 +31,25 @@
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
     & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Controls the size of the symmetric heap per PE. The value
+      set in this environment variable is interpreted as number
+      of bytes, unless the number is followed by a character that
+      acts as a multiplier, where:
+      \begin{itemize}
+        \item k or K multiplies by 2**10 (kilobytes)
+        \item m or M multiplies by 2**20 (megabytes)
+        \item g or G multiplies by 2**30 (gigabytes)
```

Comment:
What's the LaTeX rendered output for the double-star notation? Should this be guarded by math mode?

Reply:
fixed with ^^ - marking as resolved